sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/lro/AnalyzeActionsAsync.java

```diff
@@ -55,7 +55,7 @@ public static void main(String[] args) {
         client.beginAnalyzeActions(documents,
             new TextAnalyticsActions()
                 .setDisplayName("{tasks_display_name}")
-                .setExtractKeyPhrasesActions(new ExtractKeyPhrasesAction().setModelVersion("latest")),
+                .setExtractKeyPhrasesActions(new ExtractKeyPhrasesAction()),
```

Comment:
Is `setModelVersion()` no longer supported?

Reply:
The method is still supported but trying not to be specific on which model version to set in the sample case.